dvl-a50/mavlink2resthelper.py

```diff
@@ -352,12 +352,12 @@ def send_vision_position_estimate(
         )
         logger.info(post(MAVLINK2REST_URL + "/mavlink", data=data))
 
-    def send_rangefinder(self, distance: float):
+    def send_rangefinder(self, distance: float, quality: float):
         "Sends message DISTANCE_SENSOR to flight controller"
         if distance == -1:
-            return
-        data = self.rangefinder_template.format(int(distance * 100))
-
+            data = self.rangefinder_template.format(0, 0)
+        else:
+            data = self.rangefinder_template.format(int(distance * 100), int(quality))
```

Comment:
We likely need to limit the minimum quality to 1, to avoid a repeat of bluerobotics/BlueOS#1862

Reply:
Good catch @ES-Alexander !